recipes/skyfield-data/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "skyfield-data" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bae8e2d29936cae1c919ae4190aa523659043c43789631bd81de8c3b9b3815f7
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    
+test:
+  imports:
+    - skyfield_data
+    
+about:
+  home: https://github.com/brunobord/skyfield-data
+  license: MIT
+  license_family: MIT
+  license_file: 
```

Comment:
You need to specify the path to the license file here. Putting `LICENSE.txt` as the entry here should be sufficient.

Reply:
Done!